packages/material/test/renderers/MaterialCategorizationStepperLayout.test.tsx

```diff
@@ -406,4 +406,129 @@ describe('Material categorization stepper layout', () => {
+      type: 'Control',
+      scope: '#/properties/name'
+    };
+    const uischema: Categorization = {
+      type: 'Categorization',
+      label: 'Root',
+      options: {
+        showNavButtons: true
+      },
+      elements: [
+        {
+          type: 'Category',
+          label: 'B',
+          elements: [nameControl, nameControl]
+        },
+        {
+          type: 'Category',
+          label: 'C',
+          elements: undefined
+        }
+      ]
+    };
+    const store = initJsonFormsStore({
+      ...fixture,
+      uischema
+    });
+
+    const wrapper = mount(
+      <Provider store={store}>
+        <JsonFormsReduxContext>
+          <MaterialCategorizationStepperLayoutRenderer
+            {...layoutDefaultProps}
+            schema={fixture.schema}
+            uischema={uischema}
+          />
+        </JsonFormsReduxContext>
+      </Provider>
+    );
+    const prevButtonBeforeClick = wrapper.find(Button).at(1).props().disabled;
+    const nextButtonBeforeClick = wrapper.find(Button).at(0).props().disabled;
+    wrapper
+      .find(StepButton)
+      .at(1)
+      .simulate('click');
+    const prevButtonAfterClick = wrapper.find(Button).at(1).props().disabled;
+    const nextButtonAfterClick = wrapper.find(Button).at(0).props().disabled;
+
+    expect(prevButtonBeforeClick).toBe(true);
+    expect(nextButtonBeforeClick).toBe(false);
+    expect(prevButtonAfterClick).toBe(false);
+    expect(nextButtonAfterClick).toBe(true);
+    wrapper.unmount();
+  });
+
+  it('should render new category on button click', () => {
+    const nameControl: ControlElement = {
+      type: 'Control',
+      scope: '#/properties/name'
+    };
+    const innerCategorization: Categorization = {
+      type: 'Categorization',
+      label: 'Bar',
+      elements: [
+        {
+          type: 'Category',
+          label: 'A',
+          elements: [nameControl]
+        }
+      ]
+    };
+    const uischema: Categorization = {
+      type: 'Categorization',
+      label: 'Root',
+      options: {
+        showNavButtons: true
+      },
+      elements: [
+        innerCategorization,
+        {
+          type: 'Category',
+          label: 'B',
+          elements: [nameControl, nameControl]
+        },
+        {
+          type: 'Category',
+          label: 'C',
+          elements: undefined
+        },
+        {
+          type: 'Category',
+          label: 'D',
+          elements: null
+        }
+      ]
+    };
+    const store = initJsonFormsStore({
+      ...fixture,
+      uischema
+    });
+
+    const wrapper = mount(
+      <Provider store={store}>
+        <JsonFormsReduxContext>
+          <MaterialCategorizationStepperLayoutRenderer
+            {...layoutDefaultProps}
+            schema={fixture.schema}
+            uischema={uischema}
+          />
+        </JsonFormsReduxContext>
+      </Provider>
+    );
+    const beforeClick = wrapper.find(Stepper).props().activeStep;
```

Comment:
```suggestion
    const activeStepBeforeClick = wrapper.find(Stepper).props().activeStep;
```
I know that you just copied it from somewhere else in the test, but no reason to not improve on it ;)